Jenkinsfile

```diff
@@ -50,3 +53,13 @@ void mvn(def args) {
     }
   }
 }
+
+void runAthForPlugin(String plugin, String coreVersion, String testPattern) {
+  dir ('acceptance-test-harness') {
+    git 'git@github.com:jenkinsci/acceptance-test-harness.git'
```

Comment:
This should use `git://` since our build agents will not have SSH keys for GitHub access configured